tools/update-schema

```diff
@@ -219,11 +233,13 @@ def main() -> None:
     if args.force:
         print(INFO, "forcing overwrite of all schemas")
 
-    is_release = bool(os.environ.get("SCHEMA_RELEASE", False))
+    os.environ["DEV_SCHEMA"] = "" if args.prod else "1"
+    # Ensures URLs will differ based on above
+    import fmu.dataio._models as models
```

Comment:
Lazy import to ensure the models receive the `DEV_SCHEMA` env var we want them to